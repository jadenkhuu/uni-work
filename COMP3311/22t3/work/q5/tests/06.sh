python3 ./q5 Flemington 2019-11-02
