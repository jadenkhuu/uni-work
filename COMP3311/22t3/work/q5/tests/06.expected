Race meeting at Flemington on 2019-11-02

Race 1, prize pool $5000, run over 2100m
Equine Intervention ridden by Brian Hernandez Jr. wins $3500
Bill Burr ridden by Lester Piggott wins $1000
Colt Forty Five ridden by Willy Kan wins $500

Race 2, prize pool $1000, run over 1000m
Melton ridden by Paco Lopez wins $700
Pretty Polly ridden by Wally Swinburn wins $200
Dick Turpin ridden by Red Pollard wins $100

Race 3, prize pool $1000, run over 1600m
Richard Friction ridden by Josephine Gordon wins $700
Forrest Jump ridden by Emanuel Jose Sanchez wins $200
Spongebob Horsepant ridden by Rajiv Maragh wins $100

Begonia Belle Stakes, prize pool $200000, run over 1100m
Whatshesaid ridden by Colin Keane wins $140000
Gainsborough ridden by Francine Villeneuve wins $40000
Post Stallione ridden by Rosie Napravnik wins $20000

Cantala Stakes, prize pool $1000000, run over 1600m
Merry Hampton ridden by Michael Hussey wins $700000
Horsen Around ridden by Ray Cochrane wins $200000
Felstead ridden by Robbie Davis wins $100000

Carbine Club Stakes (VRC), prize pool $200000, run over 1600m
Colombo ridden by Damien Oliver wins $140000
Mane Attraction ridden by Linda Jones wins $40000
Donovan ridden by Alicia Thornton wins $20000

Coolmore Stud Stakes, prize pool $1000000, run over 1200m
Great Northern ridden by Manuel Franco wins $700000
Manna ridden by Jose Lezcano wins $200000
Horsey Kiss ridden by Alice Van-Springsteen wins $100000

Lexus Stakes, prize pool $300000, run over 2500m
St. Gatien ridden by Nina Carberry wins $210000
Blair Athol ridden by Wally Swinburn wins $60000
Nosoupforyou ridden by Peter Scudamore wins $30000

Linlithgow Stakes, prize pool $300000, run over 1200m
Alimony Pony ridden by Donna Barton Brothers wins $210000
PxHere ridden by Lester Piggott wins $60000
Rogue One ridden by Mick Dittman wins $30000

Myer Classic, prize pool $500000, run over 1600m
Long Face ridden by Robby Albarado wins $350000
Victor Wild ridden by Liam Treadwell wins $100000
Sir Hugo ridden by Manuel Franco wins $50000

Victoria Derby, prize pool $2000000, run over 2500m
Royal Lancer ridden by Hayley Turner wins $1400000
Mount Whiney ridden by Yoshitomi Shibata wins $400000
Horse Power ridden by Liam Treadwell wins $200000

Wakeful Stakes, prize pool $300000, run over 2000m
Riding Miss Daisy ridden by Irad Ortiz Jr. wins $210000
Just Mare-ied ridden by Kirsty Milczarek wins $60000
Poehler ridden by Lizzie Kelly wins $30000
